assets/jupyterLab/configure.j2.sh

```diff
@@ -11,73 +11,89 @@
 cat > requirements.txt <<-EOF
 ####
 # This is a consistent set of packages installable in Python 3, obtained by:
-# pip3 install "jupyterlab==3.0.3" "exabyte-api-client>=2020.10.19" "numpy>=1.17.3" "pandas>=1.1.4" --only-binary numpy,pandas --use-deprecated=legacy-resolver
+# pip3 install "jupyterlab==3.0.3" "exabyte-api-client>=2020.10.19" "numpy>=1.17.3" "pandas>=1.1.4"
```

Comment:
re-ran command to ensure that we actually have a consistent environment such that if we were to remove --no-deps again in future it would not break environment.